WordPress/gutenberg
# Hide the navigation block toolbar on the experimental navigation screen

## Description

Solves #21310 by hiding the navigation block toolbar as it no longer contains any useful buttons:

<img width="1576" alt="Zrzut ekranu 2020-06-9 o 13 02 50" src="https://user-images.githubusercontent.com/205419/84140207-88a7e980-aa51-11ea-863a-db6e78a6cc6b.png">

## How has this been tested?
1. Enable the navigation experiment in Gutenberg > Experiments
1. Go to Gutenberg > Navigation (beta)
1. Confirm the toolbar does not contain the alignment, navigator, colors, or an empty settings menu

## Types of changes
Non-breaking changes

## Checklist:
